includes/wizards/class-advertising-wizard.php

```diff
@@ -542,8 +553,9 @@ private function get_services() {
 		$services['google_adsense']['enabled'] = $sitekit_manager->is_module_active( 'adsense' );
 
 		// Verify GAM connection and run initial setup.
-		$gam_connection_status                   = $configuration_manager->get_gam_connection_status();
-		$services['google_ad_manager']['status'] = $gam_connection_status;
+		$gam_connection_status                               = $configuration_manager->get_gam_connection_status();
+		$services['google_ad_manager']['status']             = $gam_connection_status;
+		$services['google_ad_manager']['available_networks'] = $configuration_manager->get_gam_available_networks();
```

Comment:
`get_gam_available_networks` will throw in legacy mode, causing the Ads wizard to display the "Unrecoverable error" modal. Should be wrapped in a `try/catch` or moved down into the is-gam-connected condition.

Reply:
I figured it would be best for the returned type to be ensured at `newspack-ads`. Done in https://github.com/Automattic/newspack-ads/pull/251/commits/fc643e1c259c41d98d924b80833cbf44a274715e